src/annotator/notebook.js

```diff
@@ -34,7 +34,7 @@ export default class Notebook extends Delegator {
     this._groupId = null;
     this._prevGroupId = null;
 
-    this.container = document.createElement('div');
+    this.container = document.createElement('hypothesis-notebook');
```

Comment:
I'm the HTML validation pedant, I know: I believe this will leave us in a situation in which we wouldn't pass HTML validation; that can be problematic with a11y assessments, etc. Can any of the work we're doing to more aggressively use shadow DOM help here? This might be OK temporarily...

Reply:
> I believe this will leave us in a situation in which we wouldn't pass HTML validation; that can be problematic with a11y assessments

We are replacing a non-semantic element with one that follows the rules for custom element names, so I don't envision problems with tools that understand modern HTML. We've also been using this naming scheme (`hypothesis-xyz`) for a number of other elements for quite a while now. In the event that we do get a report of this nature I suggest we deal with it as and when it arises.

More importantly though we should make time to check that the notebook app and navigation to/from it works well with assistive tech.